bindings/rust/evmc-vm/src/container.rs

```diff
@@ -61,6 +62,20 @@ mod tests {
         }
     }
 
+    unsafe extern "C" fn get_dummy_tx_context(
+        _context: *mut evmc_sys::evmc_context,
+    ) -> evmc_sys::evmc_tx_context {
+        evmc_sys::evmc_tx_context {
+            tx_gas_price: Uint256::default(),
```

Comment:
This is ok but personally I'm not a huge fan of `Default`. Would rather have methods with descriptive names like `Uint256::zero()`, `Uint256::max()`, etc.

Reply:
Yeah, well it can be changed in a separate PR, but for tests I think it is definitely fine.